Bitcoin/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
You could add some echoes, so that we know which version we are downloading and at which stem are we.